.github/workflows/doc.yml

```
@@ -53,6 +53,27 @@ jobs:
     - name: 'Build HTML documentation'
       run: make -C Doc/ SPHINXOPTS="-q" SPHINXERRORHANDLING="-W --keep-going" html
 
+    # Add pull request annotations for Sphinx nitpicks (missing references)
+    - name: Get list of changed files
+      id: changed_files
+      uses: Ana06/get-changed-files@v2.2.0
+    - name: 'Build changed files with nitpicks'
```

I suggest using a more accurate and neutral description here., At a minimum, I suggest

```suggestion
    - name: 'Build changed files in nitpicky mode'
```

Or better yet, it would be more descriptive and IMO user-friendly (not assuming users know what nitpicky mode does) to phrase this:


```suggestion
    - name: 'Check for missing refs in changed files'
```

---

For a CI job title, this doesn’t matter much.  Sphinx calls it nitpicks, it’s a good enough keyword for someone who knows it to understand or for someone curious to search for it.